src/status/chat.nim

```diff
@@ -1,79 +1,41 @@
-import libstatus
+import core
 import json
 import utils
 
-proc startMessenger*() =
-  let payload = %* {
-    "jsonrpc": "2.0",
-    "id": 3, #TODO:
-    "method": "startMessenger".prefix,
-    "params": []
-  }
-  discard $libstatus.callPrivateRPC($payload)
-  # TODO: create template for error handling
-
 proc loadFilters*(chatId: string, oneToOne = false) =
-  let payload = %* {
-    "jsonrpc": "2.0",
-    "id": 3, #TODO:
-    "method": "loadFilters".prefix,
-    "params": [
-      [{
-        "ChatID": chatId,
-        "OneToOne": oneToOne
-      }]
-    ]
-  }
-  discard $libstatus.callPrivateRPC($payload)
+  discard callPrivateRPC("loadFilters".prefix, %* [
+    [{
+      "ChatID": chatId,
+      "OneToOne": oneToOne
+    }]
+  ])
 
 proc saveChat*(chatId: string, oneToOne = false) =
-  let payload = %* {
-    "jsonrpc": "2.0",
-    "id": 4,
-    "method": "saveChat".prefix,
-    "params": [ #TODO: determine where do these values come from
-      {
-        "lastClockValue": 0,
-        "color": "#51d0f0",
-        "name": chatId,
-        "lastMessage": nil,
-        "active": true,
-        "id": chatId,
-        "unviewedMessagesCount": 0,
-        "chatType":  if oneToOne: 1 else: 2,
-        "timestamp": 1588940692659
-      }
-    ]
-  }
-  discard $libstatus.callPrivateRPC($payload)
+  discard callPrivateRPC("saveChat".prefix, %* [
+    {
+      "lastClockValue": 0,
+      "color": "#51d0f0",
+      "name": chatId,
+      "lastMessage": nil,
+      "active": true,
+      "id": chatId,
+      "unviewedMessagesCount": 0,
+      "chatType":  if oneToOne: 1 else: 2,
```

Comment:
Not necessary to do in this PR, but we probably should replace this for an Enum, to avoid magic numbers